core/templates/pages/blog-home-page/blog-home-page.component.html

```diff
@@ -86,7 +86,7 @@
         <picture>
```

Comment:
Please add a bit more spacing using CSS as mentioned here: https://github.com/oppia/oppia/issues/19407#issuecomment-1890745921

Reply:
i have added spacing to the icon as well as no_result_found message in the line no. 46 should i make it 30 px, is it will be good ??.

Reply:
Done